Python/test/test_Closed_Field.py

```diff
@@ -52,14 +52,15 @@ def test_extended_out(self):
             = Closed_Field([1,2,2], self.date, extended_out=True)
         self.assertEqual(ans, 'LGM_CLOSED')
         numpy.testing.assert_allclose(north_fp,
-                [-0.0620421299, 0.39455411901, 0.9306480813], rtol=1e-5)
+                [-0.062047, 0.394557, 0.930652], rtol=1e-5)
```

Comment:
I presume this and other number changes are based on the new IGRF?

Reply:
Yes.